runtime/doc/lua.txt

```diff
@@ -253,6 +253,10 @@ arguments separated by " " (space) instead of "\t" (tab).
         :lua print(_VERSION)
 <    To see the LuaJIT version: >vim
         :lua =jit.version
+
+:[range]lua
+    Executes the |[range]| in the current buffer aso Lua code, regardless of
+    its extension or the current |'filetype'|.
```

Comment:
I think it's worth keeping the `|[range]|` link at least.

Reply:
Ah, no, that's not the only reason. You might want to evaluate a snippet in a Lua file as well (e.g., for simple debugging of a condition in a script).

Reply:
But you can already do that with `:[range]source`.

Reply:
Fair; in this case this is indeed useful information -- but it might be more useful to make explicit the comparison to `:source` (e.g., "unlike `:[range]source`, this evaluates the selected lines as Lua independent of `'filetype'`")?

Reply:
Yeah that's what I was thinking as well.